### PR TITLE
[coco] added stereo composer cartridge

### PR DESCRIPTION
**Pull request by tlindner**

### CODE REVIEW

File: scripts/src/bus.lua [hunk 1]
```diff
@@ -3368,6 +3368,8 @@ if (BUSES["COCO"]~=null) then
 		MAME_DIR .. "src/devices/bus/coco/coco_dcmodem.h",
 		MAME_DIR .. "src/devices/bus/coco/coco_orch90.cpp",
 		MAME_DIR .. "src/devices/bus/coco/coco_orch90.h",
+		MAME_DIR .. "src/devices/bus/coco/coco_stecomp.cpp",
+		MAME_DIR .. "src/devices/bus/coco/coco_stecomp.h",
 		MAME_DIR .. "src/devices/bus/coco/coco_ram.cpp",
```
Comment (cuavas) [inline scripts/src/bus.lua:3373]: Please keep the sources sorted here.

File: src/devices/bus/coco/coco_multi.cpp [hunk 2]
```diff
@@ -61,6 +61,7 @@
 #include "coco_fdc.h"
 #include "coco_gmc.h"
 #include "coco_orch90.h"
+#include "coco_stecomp.h"
 #include "coco_pak.h"
```
Comment (cuavas) [inline src/devices/bus/coco/coco_multi.cpp:65]: Please keep the included headers sorted within a group.

File: src/mame/drivers/dragon.cpp [hunk 3]
```diff
@@ -32,6 +32,7 @@
 #include "bus/coco/coco_ssc.h"
 #include "bus/coco/coco_ram.h"
 #include "bus/coco/coco_orch90.h"
+#include "bus/coco/coco_stecomp.h"
 #include "bus/coco/coco_gmc.h"
```
Comment (cuavas) [inline src/mame/drivers/dragon.cpp:36]: These cartridge headers have gotten mixed up as they’re added.  Please get them back in alphabetical order.

File: src/devices/bus/coco/coco_multi.cpp [hunk 4]
```diff
@@ -178,6 +180,7 @@ static void coco_cart_slot4(device_slot_interface &device)
 	device.option_add("rs232", COCO_RS232);
 	device.option_add("dcmodem", COCO_DCMODEM);
 	device.option_add("orch90", COCO_ORCH90);
+	device.option_add("stereo_composer", COCO_STEREO_COMPOSER);
```
Comment (cuavas) [inline src/devices/bus/coco/coco_multi.cpp:183]: Does the name here need to be this long?  Most of the options here use terse, abbreviated names for the command line.

**Comment by tlindner** [comment]
Should I rebase this pull request because of @aaronsgiles recent changes (the removal of voltage_ref)? Or should just submit a new pull request?

**Comment by cuavas** [comment]
Gah, when you merge upstream into your branch, GitHub no longer shows what the net effect of a merge is, making stuff like this difficult to review.

**Comment by cuavas** [comment]
If you could rebase this, I’d appreciate it.  If you don’t manage to do that in the next day or so, I’ll locally preview it client side to work out what’s going on.

**Pull request closed without merging**